### PR TITLE
chore(docs):release notes 2022.05

### LINKED ISSUE #2789: Cannot pass custom className to Accordion Content Body
I'd like to customize styling of this div https://github.com/patternfly/patternfly-react/blob/master/packages/patternfly-4/react-core/src/components/Accordion/AccordionContent.tsx#L46 but I'm not able to pass custom className there.

I wonder if that div is even necessary ? Could we just do something like

```javascript
return (
  <AccordionContentContainer
    id={id}
    className={css(
      styles.accordionExpandedContent,
      styles.accordionExpandedContentBody
      isFixed && styles.modifiers.fixed,
      !isHidden && styles.modifiers.expanded,
      className
    )}
    hidden={isHidden}
    aria-label={ariaLabel}
    {...props}
  >
    {children}
  </AccordionContentContainer>
);
```

### PR DESCRIPTION
**Pull request by tlabaj**
**What**: Closes #2789

### CODE REVIEW

File: packages/react-docs/RELEASE-NOTES.md [hunk 1]
```diff
@@ -3,6 +3,108 @@ id: Release notes
...
+- **Toolbar:**
+  - Updated to use search input instead of input group in documentation ([#7242](https://github.com/patternfly/patternfly-react/pull/7242))
+- **Wizard:**
+  - Added description component type ([#7197](https://github.com/patternfly/patternfly-react/pull/7197))
+
+### Catalog view extension
+- Updated colors to use dark theme friendly css vars ([#7215](https://github.com/patternfly/patternfly-react/pull/7215))
+
+### React styles 
+- Added Support esm import and export ([#7084](https://github.com/patternfly/patternfly-react/pull/7084))
```
Comment (evwilkin) [inline packages/react-docs/RELEASE-NOTES.md:80]: Small fix
```suggestion
- Added support for esm import and export ([#7084](https://github.com/patternfly/patternfly-react/pull/7084))
```

**Review by evwilkin — CHANGES_REQUESTED** [review]
Content looks great, just one small typo 🏆

**Review by nicolethoen — COMMENTED** [review]
@tlabaj could you also regenerate screenshots?

**Review by evwilkin — APPROVED** [review]
LGTM - thanks!

**Review by nicolethoen — APPROVED** [review]

**Pull request merged**